src/DeepLinkModule/javasource/deeplink/actions/ReadMicroflows.java

```diff
@@ -84,11 +84,16 @@ public ReadMicroflows(IContext context)
 					argumentExample.append(entry.getKey()).append("=YourValue&");
 					stringarg = true;
 				} else if( type.getType() == DataTypeEnum.Object ) {
-					IMendixObject mo = StartDeeplinkJava.query(
-					        c, Entity.getType(), Entity.MemberNames.Name, type.getObjectType());
+					List<IMendixObject> moList = Core.createXPathQuery(
+							String.format("//%s[%s=$value]", 
+								Entity.getType(),
+								Entity.MemberNames.Name))
+							.setVariable("value", type.getObjectType())
+							.execute(c);
+
                     argumentExample = new StringBuilder("YourValue&");
-                    if (mo != null) {
-                        datatype = mo;
+                    if (moList.size() > 0) {

```

Comment:
What is `mo`? Please give it a proper name, while you're at it

Reply:
Changed variable name. 
Please check and resolve :)